hoot-core/src/main/cpp/hoot/core/io/OsmXmlReader.cpp

```diff
@@ -99,7 +99,7 @@ void OsmXmlReader::_createNode(const QXmlAttributes &attributes)
   {
     newId = _map->createNextNodeId();
   }
-  LOG_VART(newId);
+  //LOG_VART(newId);
   _nodeIdMap.insert(id, newId);
```

Comment:
I commented out some of this extensive trace logging in the readers.  Its definitely useful when debugging them but makes debugging anything else difficult.